event_repository.js

```diff
@@ -0,0 +1,76 @@
+const event = require('./event')
+
+const ev1 = new event('ev1', 'This is 1st event', 'Short event 1 desciption', Date.now())
+const ev2 = new event('ev2', 'This is 2nd event', 'Description of 2nd event', Date.now())
+const ev3 = new event('ev3', 'This is last event', 'Last event for now', Date.now())
+
+let events = [ev1, ev2, ev3]
+
+function getAllEvents() {
+    return new Promise((resolve, reject) => {
+        resolve(events);
+    });
+}
+
+function getEventById(id) {
+    return new Promise((resolve, reject) => {
+        const filteredEvent = events.filter((item) => {
+            return item.id === id;
+        })[0];
```

Comment:
In this case I prefer to use the array method `find`that is more suitable when you search for just one result